packages/cli/test/unit/commands/project.test.ts

```diff
@@ -9,14 +9,17 @@ import {
   pluckIdentifiersFromDeploymentList,
   parseSpacedTableRow,
 } from '../../helpers/parse-table';
+import { useDeployment } from '../../mocks/deployment';
 
 describe('project', () => {
   describe('list', () => {
-    it('should list deployments under a user', async () => {
+    it('should list projects', async () => {
```

Comment:
**question (blocking):** Is this test working properly?

This passes in CI, but not for me locally. I see:

```
 FAIL  test/unit/commands/project.test.ts
  ● project › list › should list projects

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 1

      Array [
    -   "cli",
    +   "cli",
        "--",
      ]

      37 |         'Updated',
      38 |       ]);
    > 39 |       expect(data).toEqual([project.project.name, `--`]);
         |                    ^
      40 |     });
      41 |     it('should use inspector url', async () => {
      42 |       const user = useUser();

      at Object.<anonymous> (test/unit/commands/project.test.ts:39:20)
```

@MatthewStanciu does this pass for you locally?

Reply:
Hmm, it does pass for me locally. I can't tell what's happening to make it fail on your end.

Reply:
It turns out this was a difference between terminals that supported terminal link. In CI, terminal link is not supported. In iterm2, it is!